content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods.rst

```diff
@@ -0,0 +1,103 @@
+   :align: center
+   :alt: Reservation method field on delivery order operation type form.
+
+.. tip::
+   If the :guilabel:`Before scheduled date` reservation method is selected, a new
+   :guilabel:`Reserve before scheduled date` field appears below. From this field, the number of
+   :guilabel:`days before` and :guilabel:`days before when starred` can be changed from the default
+   `0`.
+
+   Changing the :guilabel:`days before` value changes the maximum number of days before a scheduled
+   date that products should be reserved.
+
+   Changing the :guilabel:`days before when starred` value changes the maximum number of days before
+   a scheduled date that starred (favorited) transfers for products should be reserved.
+
+   .. image:: reservation_methods/reservation-methods-before-scheduled-date.png
+      :align: center
+      :alt: Reserve before scheduled date fields with before scheduled date method chosen.
+
+Required applications
+=====================
+
+The two required applications that **must** be installed to use reservation methods are the *Sales*
+and *Inventory* apps.
+
+To install these apps, from the main Odoo dashboard, navigate to the :menuselection:`Apps`
+application, and locate the *Sales* and *Inventory* apps. Then, click :guilabel:`Install` on each
+app, and refresh the database.
+
+.. note::
+   In addition to delivery orders, reservation methods can also be used for *manufacturing orders*,
+   *resupply subcontractor* orders, orders for *repairs*, and *internal transfers*. To enable this,
+   configure the additional settings:
+
+   - **For manufacturing orders:** Install the *Manufacturing* application by going to the
+     :menuselection:`Apps` application, locating the *Manufacturing* app, and clicking
+     :guilabel:`Install`.
+   - **For resupply subcontractor:** Navigate to :menuselection:`Manufacturing app --> Configuration
+     --> Settings`, and under the :guilabel:`Operations` section, enable :guilabel:`Subcontracting`.
+     Then, click :guilabel:`Save`.
+   - **For repairs:** Install the *Repairs* application by going to the :menuselection:`Apps`
+     application, locating the *Repairs* app, and clicking :guilabel:`Install`.
+   - **For internal transfers:** Navigate to :menuselection:`Inventory app --> Configuration -->
+     Settings`, and under the :guilabel:`Warehouse` section, enable :guilabel:`Storage Locations`.
+     Then, click :guilabel:`Save`.
+
+Once these apps are installed, no additional features need to be enabled from the settings for
+reservation methods to work. They will be available by default on certain operations types, and can
+be viewed and changed by navigating to :menuselection:`Inventory app --> Configuration -->
+Operations Types`, and then clicking on a specific operations type.
+
+.. note::
+   When the :guilabel:`Type of Operation` is changed to *Receipt* on an :guilabel:`Operations Type`
+   form, reservation methods are **not** available.
```

Comment:
Please use :guilabel:s for drop-down menu options in an active instruction :)
```suggestion
   When the :guilabel:`Type of Operation` is changed to :guilabel:`Receipt` on an
   :guilabel:`Operations Type` form, reservation methods are **not** available.
```

Reply:
Suggestion implemented ✅